CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## stable-2.3.1
+This stable release adds a number of proxy stability improvements.
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Special thanks to**: @zaharidichev and @11Takanori!
+
+**Full release notes**:
+* Proxy
+  * Changed the proxy's routing behavior so that, when the control plane
+    does not resolve a destination, the proxy forwards request with minimal
+    additional routing logic
+  * Fixed a bug in the proxy's HPACK codec that could cause requests with
+    very large header values to hang indefinitely.
+  * Replaced the fixed reconnect backoff with an exponential one (thanks,
+      @zaharidichev!)
```

Comment:
You can remove 2 leading spaces from this line, to match to formatting of the other bullet points.